.github/workflows/publish-beta.yml

```diff
@@ -20,16 +20,20 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-          registry-url: 'https://registry.npmjs.org'
+          cache: 'npm'
+      - name: Check Imports
+        uses: checkdigit/github-actions/check-imports@main
       - name: Install dependencies
         run: npm ci --ignore-scripts
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          CI: true
       - name: Audit Signatures
         run: npm audit signatures
       - name: Run Test
-        run: npm run ci:test
-      - name: Prepare Beta
-        uses: checkdigit/github-actions/prepare-beta@prepare-beta
-      - name: Publish Beta
-        run: npm publish --tag beta
+        run: npm run test
+      - name: perform beta publish
+        uses: checkdigit/github-actions/publish-beta@main
```

Comment:
I think this file should be reverted.  if you just use this `uses` line from the original file it should be fine?